package.json

```diff
@@ -277,7 +277,7 @@
     "@typescript-eslint/parser": "4.18.0",
     "@wojtekmaj/enzyme-adapter-react-17": "0.6.6",
     "babel-jest": "^28.1.3",
-    "babel-loader": "8.0.6",
+    "babel-loader": "8.2.3",
```

Comment:
Node +v17 closed a security hole with SSL and I saw the following error:
```
#30 7.854 Module build failed (from ./node_modules/babel-loader/lib/index.js):
#30 7.854 Error: error:0308010C:digital envelope routines::unsupported
#30 7.854     at new Hash (node:internal/crypto/hash:71:19)
#30 7.854     at Object.createHash (node:crypto:133:10)
#30 7.854     at filename (/app/node_modules/babel-loader/lib/cache.js:94:23)
#30 7.854     at /app/node_modules/babel-loader/lib/cache.js:122:39
...
```

Upgrading `babel-loader` to [v8.2.3](https://github.com/babel/babel-loader/releases/tag/v8.2.3) resolved this.